### DISCUSSION TITLE
Help with discord notifications and "got a captcha"

### DISCUSSION THREAD

**Discussion by Worldrazor** [vogler/free-games-claimer#145] [Q&A]
Hello! 

I'm using Proxmox 7.4.3 and having the script in a LXC with Ubuntu. 

First of all I'm having trouble with getting notifications to work with discord. I'm using this command: `NOTIFY='discord://free-games-claimer@1105121665542787072/bWaZ-uYdCJ59tkAVJCmOiGdliGTAilxefyurXUnul2elumCuBX3712tGCagDrdNeekaS'` or `NOTIFY='discord://1105121665542787072/bWaZ-uYdCJ59tkAVJCmOiGdliGTAilxefyurXUnul2elumCuBX3712tGCagDrdNeekaS'`. I have checked the webhook and it works with other applications, but it seems like I need to add a username to the command? 

Secondly when I run the check for epic games I get this: 

```
2023-05-19 08:58:32.012 started checking epic-games
Not signed in anymore. Please login in the browser or here in the terminal.
Login timeout is 180 seconds!
Using email and password from environment.
Got a captcha during login (likely due to too many attempts)! You may solve it in the browser, get a new IP or try again in a few hours.
page.waitForURL: Timeout 180000ms exceeded.
=========================== logs ===========================
waiting for navigation to "https://store.epicgames.com/en-US/free-games" until "load"
============================================================
    at file:///root/free-games-claimer/epic-games.js:102:16 {
  name: 'TimeoutError'
}
```

I have changed IP already, but with same result. I have also double checked my login. 

It works great for prime gaming

**Comment by vogler** [comment]
Regarding notifications:
You're not using the Docker image? Do you have `apprise` installed? Best to test notifications directly via `apprise` and then copy the string once you got it to work. Refer to apprise's docs: https://github.com/caronc/apprise/wiki/Notify_discord

Regarding captcha:
There are several factors that may cause a captcha challenge, but for me getting a new IP was usually enough.
Check via VNC what is happening and if it really gets a captcha challenge - maybe there's a different issue.

**Reply by lslatr23** [reply]
I have the same issue. When I check via novnc, I see the captcha. and I have to solve it manually (sometimes it does not work even manually and I have to try like 10 time even when I solve the captcha correctly).

Current free game: Out of Line
  Not in library yet! Click GET.
  Got hcaptcha challenge! Lost trust due to too many login attempts? You can solve the captcha in the browser or get a new IP address.
  Failed to claim! To avoid captchas try to get a new IP address.
